mongo/options/clientoptions.go

```diff
@@ -780,9 +780,9 @@ func addCACertFromFile(cfg *tls.Config, file string) error {
 		return err
 	}
 
-	certBytes, err := loadCert(data)
+	certBytes, err := loadCACert(data)
 	if err != nil {
-		return err
+		return fmt.Errorf("error loading CA cert: %v", err)
```

Comment:
Added this error wrapping so it's clear that an error occurred when reading the CA file.